### DISCUSSION TITLE
Some clarifications regarding beta testing

### DISCUSSION THREAD

**Discussion by keiwando** [keiwando/toonsquid#625] [General]
Here are some quick clarifications since I'm currently receiving a lot of repeated questions across various different platforms and e-mail:

1. Most questions I'm receiving are already answered on the website. Please make sure to read the [ToonSquid Beta](https://toonsquid.com/beta/) page and the [FAQ](https://toonsquid.com/updates/ToonSquid-2-Beta/#faq). From the beta page:
    > New beta test invites are sent out depending on current needs. Please note that submitting this form does not guarantee if or when you will receive an invite.
    
    There will be fewer beta testers at the start and more as time goes on. There is no fixed time when invites are sent out. You are not guaranteed to receive an invite after you fill out the form. 

2. I cannot provide each person that fills out the form with a personalized explanation on whether or not they will receive an invite, when they will receive an invite or why they will not receive an invite.
3. Submitting the form multiple times does not increase your chances of receiving an invite.
5. It doesn't matter whether the e-mail address you use in the form is your Apple ID e-mail or not.
6. Don't expect the public release of this update soon. There will be a lengthy beta testing period.
7. Asking for a release date (no matter how often), will not receive a response since the question is already answered [here](https://toonsquid.com/updates/ToonSquid-2-Beta/#faq).
8. The importance of reading and understanding the [TestFlight Terms of Service](https://www.apple.com/legal/internet-services/itunes/testflight/us/terms.html) before submitting the form and participating in the beta cannot be overstated.
9. If you would like to have your form submission deleted completely, please reach out to contact@toonsquid.com from the e-mail address that you used in the form submission.

Thanks for understanding! I'm happy to see everybody's excitement for the update!

**Comment by Cramos821** [comment]
Thank you for your continued work on toon squid. Even though it will be a while before 2.0 releases, it already seems incredible!

**Comment by DWrightAnimation** [comment]


**Reply by Cramos821** [reply]
Please don’t be rude.

**Reply by CowlBlue** [reply]
yea there is no reason to be so aggressive.

**Reply by joelmayer87** [reply]
I swear the dev probably had to make this post solely because of this guy…

**Comment by rdc457** [comment]
Thank you for ur clarification!

**Comment by TempleLionStudios** [comment]
I remember reading somewhere that using the beta could resolve in corrupting projects? Like a warning if your already use toonsquid? But now I can't find that info anywhere... anyone know anything? I have a years work in regular Toonsquid already, that I don't wanna lose!

**Reply by Lioncub901** [reply]
Got the beta. I don't use any of my personal stuff. I just import them just to be safe. I would not recommend relying on beta software.  Hopes that helps.

**Reply by Cat-Butler** [reply]
Sounds like this comment and replies: https://github.com/keiwando/toonsquid/discussions/624#discussioncomment-10142031

Summary: some app betas replace the original app, which can result in possible problems during testing as new features affect the file format. Keiwan has made the beta a separate app to avoid the problems. 

However, as @Lioncub901 indicates, beta file formats will possibly be different, so only work with new projects in the beta, or projects you can afford to lose. Once you have opened and saved a project in the beta, there is no guarantee it will re-open in 1.x, or even (if you are paranoid enough) in the final release version
